sdk/src/transaction_builder.rs

```diff
@@ -51,6 +50,20 @@ impl TransactionBuilder {
         }
     }
 
+    /// Create a new unsigned transaction from a single instruction
+    pub fn new_singleton(instruction: BuilderInstruction) -> Transaction {
```

Comment:
How about we dumb this down as `new_with_instruction()`?  The word "singleton" brings me directly to thinking about the Singleton pattern, which this is not.

Reply:
done